### PR TITLE
discovery: Add basic language detection

### PR DESCRIPTION
**Pull request by vitkyrka**
### What does this PR do?

Add basic language detection based only on the command line of the service.  More elaborate language detection based on reading the binary of the process can be enabled later after more evaluation.

### CODE REVIEW

File: pkg/collector/corechecks/servicediscovery/impl_linux_test.go [hunk 1]
```diff
@@ -59,6 +59,15 @@ var (
 			Starttime: procLaunchedClicks,
 		},
 	}
+	procPythonService = testProc{
```
Comment (guyarb) [inline pkg/collector/corechecks/servicediscovery/impl_linux_test.go:62]: let's add tests that are not covered by the current language detection algorithm

**Review by guyarb — APPROVED** [review]

File: pkg/collector/corechecks/servicediscovery/impl_linux_test.go (hunk 1 above)
Comment (vitkyrka) [reply pkg/collector/corechecks/servicediscovery/impl_linux_test.go:62]: The other existing tests cover the case where the language detection algorithm fails.

Comment (vitkyrka) [reply pkg/collector/corechecks/servicediscovery/impl_linux_test.go:62]: The other existing tests cover the case where the language detection algorithm fails.

**Pull request merged**